.github/workflows/mvn-verify-check.yml

```
@@ -75,9 +75,10 @@ jobs:
           distribution: adopt
           java-version: 8
 
-      - name: package aggregator check
+      - name: package tests check
         run: >
-          mvn -Dmaven.wagon.http.retryHandler.count=3 -B package -pl aggregator -am
+          mvn -Dmaven.wagon.http.retryHandler.count=3 -B package
+          -pl integration_tests -am -pl tests -am
```

repeated  use `-pl` - even if this works this looks unconventional to me https://maven.apache.org/ref/3.6.3/maven-embedder/cli.html. It should be a comma-separated list 
```suggestion
          -pl integration_tests,tests -am
```